backend/routes/projectById.js

```diff
@@ -0,0 +1,34 @@
+const { parse } = require("url");
+
+const { missingProperty, invalidProperty } = require('../src/utils/errors');
+const stringify = require('../src/utils/stringify');
+const getAllProjects = require('../src/components/projects/getAllProjects');
+
+module.exports = async (req, res) => {
+  const { query: { project: projectParam } } = parse(req.url, true);
```

Comment:
projectParam, I am not sure what this name means. I'd expect that this is some general purpose param that gets passed to a project, but also it can be understood as the name of the project. Better name pls.

Reply:
'Cause ambiguity sucks